recipes/sasl/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "sasl" %}
+{% set version = "0.2.1" %}
+{% set md5 = "ca093d9a3d6f20b79b964a5e5add0202" %}
```

Comment:
Please use the `sha256` hash:

https://conda-forge.org/docs/meta.html#populating-the-hash-field